src/CharacterState.js

```diff
@@ -60,6 +60,7 @@ export default class CharacterState {
         return true;
     }
 
+    // eslint-disable-next-line no-use-before-define
```

Comment:
Are there any circumstances in which this check is helpful to us?  If not, maybe we should disable the rule rather than adding all these comments.

Reply:
That's a good question. Disabling the rule, would certainly make the code cleaner by not requiring there comments.

I think the rule offers some value but I think not a huge amount. I'll disable the rule and remove the comments. We can always reconsider. But I think minimizing the number of ESLint-specific comments is worthwhile.

Reply:
I've disabled the "no-use-before-define" rule